scheduler/src/cook/test/testutil.clj

```diff
@@ -543,14 +555,14 @@
   node))
 
 (defn make-kubernetes-compute-cluster
-  [namespaced-pod-name->pod pool-names synthetic-pods-user node-blocklist-labels]
+  [cluster-entity-id namespaced-pod-name->pod pool-names synthetic-pods-user]
```

Comment:
In your refactors cook.test.scheduler.test, I still have a feeling that you don't need to make this change, and just need to with-redef the function that thats creating the NPE with a mock.  Lets keep this comment open and see if we can revert this change.

Reply:
Used a few `reify`s to mock out at the compute-cluster level, so I reverted this change.